src/components/Jobs/components/PreviewsJobs/PreviewsJobs.js

```diff
@@ -8,21 +8,21 @@ import {
   PROCESSING_FINISHED,
   PROCESSING_IN_PROGRESS,
 } from '../../jobStatuses';
-import { DataFetcherContext } from '../../../DataFetcher/DataFetcherContext';
+import { DataFetcherContext } from '../../../DataFetcher';
 
 class PreviewsJobs extends PureComponent {
   static contextType = DataFetcherContext;
 
   prepareJobsData() {
     const jobStatuses = [PROCESSING_FINISHED, PROCESSING_IN_PROGRESS]; // TODO: could be changed on backend
-    const jobs = get(this.context, ['jobs', 'itemsObject', 'jobExecutionDtos'], [])
+    const jobs = get(this.context, ['jobs', 'jobExecutionDtos'], [])
       .filter(({ status }) => jobStatuses.includes(status));
 
     return sortPreviewJobs(jobs);
   }
 
   render() {
-    const hasLoaded = get(this.context, ['jobs', 'hasLoaded'], false);
+    const { hasLoaded = false } = this.context;
```

Comment:
Is default value needed? Why it cannot be retrieved right away from context as is?